src/components/organisms/EntityDetails/EntityDetailsContent/SummaryGrid/SummaryGrid.styled.tsx

```diff
@@ -1,18 +1,40 @@
 import styled from 'styled-components';
 
+import {StyledText} from '@custom-antd/Typography/Text/Text.styled';
+
 import Colors from '@styles/Colors';
 
 export const SummaryGridWrapper = styled.div`
   display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(235px, 1fr));
   gap: 12px;
+  grid-template-columns: 1fr;
+
+  @media (min-width: 507px) {
+    grid-template-columns: 1fr 1fr;
+  }
+
+  @media (min-width: 860px) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @media (min-width: 1112px) {
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  }
+`;
```

Comment:
we might want to abstract these breakpoints one day – but not necessary now...

Reply:
Yeah, fully aware of it. I guess more general solution will be needed.